ably-ios/ARTPresenceMessage.m

```diff
@@ -34,4 +34,8 @@ - (NSString *)description {
     return description;
 }
 
+- (NSString *)memberKey {
+    return [NSString stringWithFormat:@"%@:%@", self.connectionId, self.clientId];
```

Comment:
In `PresenceMessage` both `connectionId` and `clientId` are defined as non-nullable, so that's something we don't have to worry about.